src/sails_forth/batch.clj

```diff
@@ -0,0 +1,30 @@
+(ns sails-forth.batch
+  (:require [clojure.spec.alpha :as s]
+            [sails-forth.http :as http]
+            [sails-forth.spec :as spec]))
+
+(s/def ::batch-response (s/coll-of map?))
+
+(s/fdef batch!
+        :args (s/cat :client ::http/client
+                     :type ::spec/type
+                     :attrs ::spec/attrs)
+        :ret ::batch-response)
+
+(defn batch!
+  [client type reqs]
```

Comment:
What is the point to have the type passed in here if it's always going to be "batch"?